src/services/identity/ReposService.ts

```diff
@@ -151,14 +151,21 @@ export default class ReposService {
     await this.simpleGit.cwd({ path: dir, root: false }).checkout("staging")
   }
 
-  private setUrlsInLocalConfig(
+  private async setUrlsInLocalConfig(
     dir: string,
     repoName: string,
     stagingUrl: string,
     productionUrl: string
   ) {
     const configPath = `${dir}/_config.yml`
-    const configFile = fs.readFileSync(configPath, "utf-8")
+    let configFile: string
+    try {
+      configFile = await fs.promises.readFile(configPath, "utf-8")
+    } catch (error) {
+      throw new UnprocessableError(
+        `Error reading _config.yml for '${repoName}': ${error}`
+      )
+    }
```

Comment:
ok with the `throw` but how do we return something useful back to our user?

Reply:
hmm i am a bit confused by on what you refer to as useful.
this is called in the site create side form, and we return the error as is to the us, the admins of isomer. 

looking at the rest of of the calls, they seem to return something similar, inclined to keep the local consistent behaviour of the throws for this file

Reply:
Ok I thought this was user facing. If it doesn't cause the flow to just stop here without a useful error message then I'm ok w just throwing.